deployment/main.tf

```diff
@@ -0,0 +1,153 @@
+#
+# Terraform file for deploying lnt.llvm.org.
+#
+
+variable "lnt_db_password" {
+  type        = string
+  description = "The database password for the lnt.llvm.org database."
+  sensitive   = true
+}
+
+variable "lnt_auth_token" {
+  type        = string
+  description = "The authentication token to perform destructive operations on lnt.llvm.org."
+  sensitive   = true
+}
+
+locals {
+  # The Docker image to use for the webserver part of the LNT service
+  lnt_image     = "d9ffa5317a9a42a1d2fa337cba97ec51d931f391"
+
+  # The port on the EC2 instance used by the Docker webserver for communication
+  lnt_host_port = "80"
+}
+
+terraform {
+  backend "s3" {
+    bucket  = "lnt.llvm.org-test-bucket" # TODO: Adjust this for the real LLVM Foundation account
```

Comment:
Is the plan to also control this bucket within terraform?

Reply:
I was mainly thinking of what we do in premerge where we also create the bucket in terraform.

https://github.com/llvm/llvm-zorg/blob/a811e7962f335663ccc123026839bdc35409d395/premerge/main.tf#L18

If it's easier to create manually, we can probably just do that.